crawler/config.py

```diff
@@ -9,6 +9,36 @@ class RSSConfig:
     links = {
         "The Verge": "https://www.theverge.com/rss/index.xml",
         "Engadget": "https://www.engadget.com/rss.xml",
+        "Life Hacker": "https://lifehacker.com/rss",
+        "Wired": "https://www.wired.com/feed/rss",
         # Feedburner based XML Feed, need to be careful
         "TechCrunch": "http://feeds.feedburner.com/TechCrunch/"
+    }
+
+    item_name = {
+        "The Verge": "entry",
```

Comment:
Where do we consume this mapping? We seem to have a lot of string-to-string mapping, where content provider names like "The Verge" is used in multiple structures (item_name, item_publish)...